cwf/gateway/fabfile.py

```diff
@@ -45,11 +60,13 @@ def list():
         return list(map(lambda t: t.value, SubTests))
 
 
-def integ_test(gateway_host=None, test_host=None, trf_host=None,
-               gateway_vm="cwag", gateway_ansible_file="cwag_dev.yml",
-               transfer_images=False, destroy_vm=False, no_build=False,
-               tests_to_run="all", skip_unit_tests=False, test_re=None,
-               test_result_xml=None, run_tests=True, count="1"):
+def integ_test(
+    gateway_host=None, test_host=None, trf_host=None,
+    gateway_vm="cwag", gateway_ansible_file="cwag_dev.yml",
+    transfer_images=False, destroy_vm=False, no_build=False,
+    tests_to_run="all", skip_unit_tests=False, test_re=None,
+    test_result_xml=None, run_tests=True, count="1", force_provision=True,
+):
```

Comment:
what about we use `provision_vm` or we change lte_integ_test to `force_provision` for consistency?

Reply:
ops just saw it is merged :D

Reply:
ya i can change in a separate PR :)